### PR TITLE
DevKit updates for 4.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

**Comment by jordisala1991** [comment]
@franmomu do you know if the next minor of doctrine odm bundle is happening soon? otherwise we will have to add special case for this bundle, since it is the only one that cant install the latest php cs fixer, due to doctrine annotations being locked on 1.x

**Comment by franmomu** [comment]
> @franmomu do you know if the next minor of doctrine odm bundle is happening soon? otherwise we will have to add special case for this bundle, since it is the only one that cant install the latest php cs fixer, due to doctrine annotations being locked on 1.x

I think it shouldn't take long, I'll try to push things forward

File: composer.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     },
     "require-dev": {
         "doctrine/data-fixtures": "^1.4",
-        "friendsofphp/php-cs-fixer": "^3.4",
+        "friendsofphp/php-cs-fixer": "^3.15",
```
Comment (jordisala1991) [inline composer.json]:
```suggestion
        "friendsofphp/php-cs-fixer": "^3.4",
```

File: .github/workflows/lint.yaml [hunk 2]
```diff
@@ -29,7 +29,7 @@ jobs:
             - name: Install PHP with extensions
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: '8.1'
+                  php-version: '8.2'
```
Comment (jordisala1991) [inline .github/workflows/lint.yaml]:
```suggestion
                  php-version: '8.1'
```

**Comment by jordisala1991** [comment]
Lets unblock this Pr by changing back to 8.1 to fix the build. Dev kit will open another Pr with only that change

**Pull request merged**